apps/browser/src/tools/popup/send-v2/send-v2.component.html

```diff
@@ -6,11 +6,11 @@
       <app-current-account></app-current-account>
     </ng-container>
   </popup-header>
-  <div slot="above-scroll-area" class="tw-p-4">
+  <div slot="above-scroll-area" [ngClass]="{ 'tw-p-4': (sends$ | async)?.length }">
```

Comment:
Testing with the solution below and my other suggestion I haven't experienced any issues with added padding or mis-rendering.

```suggestion
  <div slot="above-scroll-area" class="tw-p-4">
```